### PR TITLE
[#1675] blank edit drep when invalid data

### PR DESCRIPTION
**Pull request by jdyczka**
## List of changes

- Fix runtime error at /edit_drep when metadata incorrect

## Checklist

- [related issue](https://github.com/IntersectMBO/govtool/issues/1675)
- [x] My changes generate no new warnings
- [x] My code follows the [style guidelines](https://github.com/IntersectMBO/govtool/tree/main/docs/style-guides) of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the [changelog](https://github.com/IntersectMBO/govtool/blob/main/CHANGELOG.md)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: govtool/frontend/src/components/molecules/GovernanceActionsDatesBox.tsx [hunk 1]
```diff
@@ -4,11 +4,12 @@ import { Trans } from "react-i18next";
 
 import { Tooltip, Typography } from "@atoms";
 import { useScreenDimension, useTranslation } from "@hooks";
+import { formatDisplayDate } from "@/utils";
 
 type GovernanceActionsDatesBoxProps = {
   createdDate: string;
-  expiryDate: string;
-  expiryEpochNo: number;
+  expiryDate: string | undefined;
```
Comment (MSzalowski) [inline govtool/frontend/src/components/molecules/GovernanceActionsDatesBox.tsx:11]:
```suggestion
  expiryDate?: string;
```

**Review by MSzalowski — CHANGES_REQUESTED** [review]
Fix checks :saluting_face:

File: govtool/frontend/src/components/molecules/GovernanceActionsDatesBox.tsx (hunk 1 above)
Comment (jdyczka) [reply govtool/frontend/src/components/molecules/GovernanceActionsDatesBox.tsx:11]: Right, but that way the `expiryDate` prop would not be required. And I think it's better if it's required but accepts undefined just for the types to match

Comment (MSzalowski) [reply govtool/frontend/src/components/molecules/GovernanceActionsDatesBox.tsx:11]: Agree with that

**Pull request merged**